apis/ec2/v1alpha1/zz_vpc_peering_connection.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// VPCPeeringConnectionParameters defines the desired state of VPCPeeringConnection
+type VPCPeeringConnectionParameters struct {
+	// Region is which region the VPCPeeringConnection will be created.
+	// +kubebuilder:validation:Required
+	// +immutable
+	Region string `json:"region"`
+	// Checks whether you have the required permissions for the action, without
+	// actually making the request, and provides an error response. If you have
+	// the required permissions, the error response is DryRunOperation. Otherwise,
+	// it is UnauthorizedOperation.
+	DryRun *bool `json:"dryRun,omitempty"`
```

Comment:
```suggestion
```

This isn't really applicable in a CRD, so we should make code-generator ignore this field via `generator-config.yaml`